device-types/Arista/AWE-5510.yaml

```diff
@@ -0,0 +1,53 @@
+---
+manufacturer: Arista
+model: AWE-5510
+slug: arista-awe-5510
+part_number: AWE-5510
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c16
+    maximum_draw: 800
+  - name: PS2
+    type: iec-60320-c16
+    maximum_draw: 800
```

Comment:
It appears that these PSUs are modular. If this is the case, then please move the power to `module-bays` and create the associated modular PSUs within the respective module-types folder